draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
I thought we were just talking about in initial offers; in subsequent offers, the "a=msid" line should stay the same, since according to mmusic-msid, removing it will cause the track to end:
https://tools.ietf.org/html/draft-ietf-mmusic-msid-16#section-3.2.5